src/main/scala/mesosphere/marathon/core/deployment/ScalingProposition.scala

```diff
@@ -30,7 +34,7 @@ object ScalingProposition extends StrictLogging {
         toDecommissionMap.contains(instanceId)
     }
     // overall number of tasks that need to be killed
-    val decommissionCount = math.max(instancesGoalRunning.size - scaleTo, sentencedAndRunningMap.size)
+    val decommissionCount = math.max(instancesRunning.size - scaleTo, sentencedAndRunningMap.size)
```

Comment:
Do you think there could be side effect of this ?

Reply:
See previous comment :-)